mention/CHANGELOG.md

```diff
@@ -2,6 +2,15 @@
 
 Changelog for `twilight-mention`.
 
+## [0.4.2] - 2021-06-12
+
+### Changes
+
+The mention implementation has been made more generic. There are no breaking changes ([#883] -
```

Comment:
I said earlier that refactoring PRs aren't mentioned, but since this is a new contributor I personally prefer to mention their contributions. This can continue to be mentioned IMO